hack/vendor.sh

```diff
@@ -24,5 +24,5 @@ if [ $# -eq 0 ] || [ "$1" = "archive/tar" ]; then
 fi
 
 if [ $# -eq 0 ] || [ "$1" != "archive/tar" ]; then
-	vndr -whitelist=^archive/tar "$@"
+	vndr -whitelist='^archive[/\\]tar' "$@"
```

Comment:
Interesting; it's not matching on package name but on directory? (I assume this was failing on Windows?)

Reply:
Yeah, it looks like it's using the whitelist in two places:
- `filepath.Walk` fed into checking the whitelist: https://github.com/LK4D4/vndr/blob/master/clean.go#L84-L98
- `ioutil.ReadDir` fed into checking the whitelist: https://github.com/LK4D4/vndr/blob/master/clean.go#L142-L154

So sadly not a package path :(